Library/Homebrew/cask/auditor.rb

```diff
@@ -26,7 +26,6 @@ def initialize(
       quarantine: nil,
       any_named_args: nil,
       language: nil,
-      display_passes: nil,
       display_failures_only: nil,
```

Comment:
```suggestion
```

will presumably be able to go away. 

I'd suggest having a `strict` here somewhere, too.

Reply:
`@audit_strict` already exists.